contributors/guide/pull-requests.md

```diff
@@ -590,6 +590,37 @@ at once to that file.
 * Can the file be improved further?
 * Does the trivial edit greatly improve the quality of the content?
 
+## Fixing linter issues
+
+Kubernetes has a set of linter checks. Some of those must pass in the entire
+code base, some must pass in new or modified code, and some are merely hints
+to developers how to improve their code.
+
+Please do not create Pull Requests for issues found by linters without first
+reaching out to maintainers on the `#code-organization`
+[Slack](http://slack.kubernetes.io) channel to determine whether there is
+sufficient interest in fixing such issues.
+
```

Comment:
```suggestion
When it was discussed, make sure to include people who gave the preliminary approval of this work as well as the link to the discussion on Slack or github issue into the PR description. Example may be:


> This PR fixes linter rules discussed in the Slack https://kubernetes.slack.com/archives/Foo/Bar.
> Preliminary agreement to address those issues were given by @GHHandle1 and @GHHandle2.
>
> /assign @GHHandle1
> /assign @GHHandle2
>
> This PR fixes issues in the package:
> pkg/kubelet
>
> Related PRs for other packages:
> - github.com/link-to-other-PR1
> - github.com/link-to-other-PR2
>

```

Reply:
Since this is mostly done by drive-by contributor, an explicit template will be helpful

Reply:
@SergeyKanzhelev: perhaps you can add back the LGTM if you are happy with it now?

Reply:
thank you!
/lgtm